app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -74,10 +73,112 @@ class SubjectAreaDropdown extends React.Component {
+      }),
+      placeholder: (base, state) => ({
+        ...base,
+        color: this.props.theme.colorText,
+        margin: `${gridUnitValue / 2}px`,
+        padding: `${gridUnitValue / 2}px`,
+      }),
+      input: (base, state) => ({
+        ...base,
+        padding: `${gridUnitValue / 2}px`,
+      }),
       multiValue: (base, state) => ({
         ...base,
         backgroundColor: this.props.theme.colorPrimary,
+        color: this.props.theme.colorTextReverse,
+        margin: `${gridUnitValue / 2}px`, // combines with padding on valueContainer to achieve gridUnit spacing around tags
+      }),
+      multiValueLabel: (base, state) => ({
+        ...base,
+        color: this.props.theme.colorTextReverse,
+        fontSize: this.props.theme.fontSizeBase,
+        padding: '8px',
+        paddingLeft: '8px',
+      }),
+      multiValueRemove: (base, state) => ({
+        ...base,
+        padding: `0 ${this.props.theme.gridUnit} 0 0`,
+        backgroundColor: this.props.theme.colorPrimary,
+        ':hover': {
+          cursor: 'pointer',
+          // overriding react-select's default hover behaviour - changing colours
+          backgroundColor: this.props.theme.colorPrimary,
+          color: this.props.theme.colorTextReverse,
+        },
+      }),
+      menuList: (base, state) => ({
+        ...base,
+        padding: 0,
+      }),
+      menu: (base, { placement }) => ({
+        ...base,
+        borderRadius: `0 0 ${this.props.theme.borderRadius} ${
+          this.props.theme.borderRadius
+        }`,
+        borderWidth: this.props.theme.borderWidth,
+        marginBottom: 0,
+        // leave room for the bottom border of the Control component to be visible on validation/focus
+        marginTop: this.props.theme.borderWidth,
+      }),
+      option: (base, { isSelected, isFocused }) => ({
+        ...base,
+        backgroundColor:
+          isSelected || isFocused
+            ? this.props.theme.colorPrimary
+            : 'transparent',
+        color:
+          isSelected || isFocused
+            ? this.props.theme.colorTextReverse
+            : this.props.theme.colorText,
+        padding: `${gridUnitValue * 2}px`,
+      }),
+      control: (base, { isFocused }) => ({
+        ...base,
+        borderRadius: this.props.theme.borderRadius,
+        borderWidth: this.props.theme.borderWidth,
+        borderStyle: this.props.theme.borderStyle,
+        borderColor: chooseBorderColorFromProps(
+          this.props.theme,
+          this.props.validationStatus,
+          isFocused,
+        ),
+        // overrides the fact that react-select changes borderColor on hover
+        '&:hover': {
+          borderColor: chooseBorderColorFromProps(
+            this.props.theme,
+            this.props.validationStatus,
+            isFocused,
+          ),
+        },
+        boxShadow: isFocused
```

Comment:
The logic for this feels really convoluted but wasn't sure how else to achieve it. Suggestions welcome.

(I added this since your review btw @tamlyn)

Reply:
Didn't see that this had been done in other components before. Should I be following [the pattern used on the checkbox](https://gitlab.coko.foundation/pubsweet/pubsweet/blob/master/packages/ui/src/atoms/Checkbox.js#L74)?

Reply:
Implementation looks good but we need more input on styling from Nick